refresh.template.py

```diff
@@ -979,20 +979,52 @@ def _get_commands(target: str, flags: str):
 
 
     # Then, actually query Bazel's compile actions for that configured target
-    target_statement = f'deps({target})' # TODO we should always be quoting targets when we splice them in. Let's use single quotes like with mnemonic, above. See slightly down in https://bazel.build/query/language#tokens
+    target_statement = f"deps('{target}')"
```

Comment:
Minor, but as in the TODO, we should really be quoting targets everywhere!

Reply:
I also quote in other queries. But I found another problem in bazel query.
Because of the bug mentioned above I tried to use another query statement to return the middle_target_candidate.
`f"filter({re.escape(candidate}$)"` but when the label starts with `@` it will return empty realist... so I just remove the first @ symbol for workaround https://github.com/xinzhengzhang/bazel-compile-commands-extractor/pull/1/commits/6b098e83ea8c1d1218c8afca555b445dbdcd6b5d#diff-a1d7061df7c566a1f7656624ec608ad53dd3aff7a7789b9b6e4866a3b1616042R1016